### PR TITLE
Added hasReference in schema and example + reference to eForms fields

### PR DESCRIPTION
**Pull request by ColinMaudry**
And changed the title of the extension from "Recurring contracts" to "Recurrence" to match the repo's name and titles of the fields.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,16 @@ This information can be useful, for example, to companies deciding whether to in
...
 
-The European Union is a [party](https://www.wto.org/english/tratop_e/gproc_e/memobs_e.htm) to the GPA, and as such its [Directive 2014/24/EU](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2014.094.01.0065.01.ENG) (Public contracts — setting out clear ground rules) includes: "Part C: Information to be included in contract notices … 27. In the case of recurrent procurement, estimated timing for further notices to be published."
+The European Union is a [party](https://www.wto.org/english/tratop_e/gproc_e/memobs_e.htm) to the GPA, and as such its [Directive 2014/24/EU](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2014.094.01.0065.01.ENG) (Public contracts — setting out clear ground rules) includes: "Part C: Information to be included in contract notices … 27. In the case of recurrent procurement, estimated timing for further notices to be published".
```
Comment (jpmckinney) [inline README.md]: Keep the period inside the quotation marks (American English style).

File: README.md [hunk 2]
```diff
@@ -12,13 +12,16 @@ This information can be useful, for example, to companies deciding whether to in
...
+
+This extension's field correspond to [eForms BT-94 and BT-95](https://github.com/eForms/eForms).
```
Comment (jpmckinney) [inline README.md]: Can we put the names of the BTs?

**Review by jpmckinney — COMMENTED** [review]
A few small comments. Otherwise all good!

File: README.md (hunk 1 above)
Comment (ColinMaudry) [reply README.md]: These Americans... Done!

File: README.md (hunk 2 above)
Comment (ColinMaudry) [reply README.md]: Done

**Pull request merged**